overlord/state/export_test.go

```diff
@@ -32,4 +32,8 @@ func ChangeUnlockCheckpointRetryParamsForTest(newInterval, newMaxTime time.Durat
 	return
 }
 
+func (r *TaskRunner) Handlers() map[string]HandlerFunc {
```

Comment:
It could also just be a method in TaskRunner... seems simple and obvious enough, even if not very useful outside of tests yet.